src/client/helpers/improve.js

```diff
@@ -0,0 +1,8 @@
+const latexRegex = /\[\+\]((\n|.)*?)\[\+\]/g;
```

Comment:
This could be in `regexHelpers` but I believe it makes sense to separate regexes used for improve engine from other ones. This might become it's own package in the future, so it makes sense to leave it there.